scripts/kickstart_generator.rb

```diff
@@ -23,17 +23,17 @@ def initialize(build_base, targets, puddle, manageiq_checkout, appliance_checkou
       @ssui_checkout      = ssui_checkout
     end
 
-    def run(task = :all)
+    def run
+      FileUtils.mkdir_p(@ks_gen_base)
+
       targets.each do |target|
         @target = target # used during ERB evaluation
 
         result = evaluate_erb
 
-        FileUtils.mkdir_p(@ks_gen_base)
-        File.write(@ks_gen_base.join("base.ks"), result)
-
-        write_config(result) if [:all, :config].include?(task)
-        write_json(result)   if [:all, :json].include?(task)
+        file = @ks_gen_base.join("base-#{@target}.ks")
+        $log.info("Writing kickstart file: #{file}") if $log
+        File.write(file, result)
```

Comment:
We needed config and json format at one point, while it's worthwhile checking if we still need it, we shouldn't change this as part of this feature.  

Note, I believe imagefactory/koji at one point required one type of file as input and then needed a different format.  If we only ever use one format for both imagefactory/koji builds, I'm good with removing the "unused" format in a separate PR.  CC @simaishi @carbonin